### PR TITLE
fix(#679): ensure that ids are unique when copying components

### LINKED ISSUE #679: Copying the same element after refreshing the page will result in the same id
**Version**
@sunmao-ui/runtime: "^0.8.0",
@sunmao-ui/editor: "^0.8.0",

**Steps to reproduce**
1. Create an element with slots(e.g. `Stack`)
2. Insert an element and copy it, behaves normally
3. Refreshing the page and copying it again will result in two elements with the same id, which will cause the page to crash
copy_bug

**What is expected?**
When copied again, the same id should not appear

### PR DESCRIPTION
**Pull request by xzdry**
resolve #679

### CODE REVIEW

File: packages/editor/src/operations/leaf/component/pasteComponentLeafOperation.ts [hunk 1]
```diff
@@ -9,16 +9,19 @@ export type PasteComponentLeafOperationContext = {
...
   private componentCopy!: IComponentModel;
 
   do(prev: AppModel): AppModel {
+    let copyTimes = 0;
+    const ids = prev.allComponents.map(c => c.id);
     this.context.component.allComponents.forEach(c => {
-      c.changeId(`${c.id}_copy${copyTimes}` as ComponentId);
+      let copyId = `${c.id}_copy${copyTimes}`;
+      while (ids.includes(copyId as ComponentId)) {
+        copyId = `${c.id}_copy${++copyTimes}`;
```
Comment (tanbowensg) [inline packages/editor/src/operations/leaf/component/pasteComponentLeafOperation.ts]: Don't use `++copyTimes`. It's hard to read.

File: packages/editor/src/operations/leaf/component/pasteComponentLeafOperation.ts [hunk 2]
```diff
@@ -9,16 +9,19 @@ export type PasteComponentLeafOperationContext = {
   component: IComponentModel;
 };
 
-let copyTimes = 0;
-
 export class PasteComponentLeafOperation extends BaseLeafOperation<PasteComponentLeafOperationContext> {
   private componentCopy!: IComponentModel;
 
   do(prev: AppModel): AppModel {
+    let copyTimes = 0;
```
Comment (tanbowensg) [inline packages/editor/src/operations/leaf/component/pasteComponentLeafOperation.ts:16]: If you move `copyTimes` to local scope, copyTimes will start from 0 every time user pastes. `copyTimes` should be global.

Comment (xzdry) [reply packages/editor/src/operations/leaf/component/pasteComponentLeafOperation.ts:16]: Shared `copyTime` between components can cause copyTime to grow too fast and looks strangely     
For example，copy stack5 after copying text several times, the copytime becomes 13
image

File: packages/editor/src/operations/leaf/component/pasteComponentLeafOperation.ts (hunk 1 above)
Comment (xzdry) [reply packages/editor/src/operations/leaf/component/pasteComponentLeafOperation.ts]: changed

**Comment by xzdry** [comment]
Added test, PTAL

**Review by tanbowensg — APPROVED** [review]

**Pull request merged**